app.py

```diff
@@ -1,10 +1,146 @@
-from flask import Flask
+from flask import Flask, jsonify, request
+import traceback
+import random
+import datetime
+from test_mongo_data import Mdb
+from config import ITEM, CONTAINER, ARRANGEMENT, SNAPSHOT
+
 app = Flask(__name__)
+mdb = Mdb()
+
+def uniqueid():
+    seed = random.getrandbits(32)
+    while True:
+       yield seed
+       seed += 1
 
 
-@app.route('/')
-def hello():
+@app.route("/")
+def home_page():
     return "Arrange That!"
 
+@app.route('/arrangement', methods=['POST'])
+@app.route('/api/v1/arrangement', methods=['POST'])
+def add_arrangement():
+    content = request.json
+    try:
+        data = {}
```

Comment:
data should just equal content. There should be no content creation done in the backend. The backend is just a data store.

data = content.

No need for add_item or add_container.

I would just have a method that we can create later called verify_arrangement, that we pass content into and that will verify that it is a well formed object.